instructure/canvas-ios
# Add trailing item groups sync to helper method.

refs: MBL-16657
affects: Teacher
release note: Fixed assignment edit button not being visible on module page.

test plan:
- Go to course > modules list
- Select an assignment
- Check if edit button is visible on the top right

## Screenshots

<table>
<tr><th>Before</th><th>After</th></tr>
<tr>
<td><img src="https://user-images.githubusercontent.com/72396990/234264534-9c8d230b-67af-4880-aec2-550f39767b44.png" maxHeight=500></td>
<td><img src="https://user-images.githubusercontent.com/72396990/234264527-d4df2dcd-352d-40e8-83a8-f06036faf663.png" maxHeight=500></td>
</tr>
</table>

## Checklist

- [ ] Tested on phone, iOS 15
- [ ] Tested on tablet, iOS 15
- [ ] Tested on phone, iOS 16
- [ ] Tested on tablet, iOS 16
